prow/cmd/phony/README.md

```diff
@@ -0,0 +1,33 @@
+# Phony
+
+`phony` sends fake GitHub webhooks.
+
+## Running a GitHub event manager
+`phony` is most commonly used for testing [hook](../hook), but can be used for testing any
+externally exposed service configured to receive GitHub events.
+
+To get an idea of `phony`'s behavior, start a local instance of `hook` with
+this:
+```
+go run prow/cmd/hook/main.go --deck-url=localhost
+--config-path=prow/config.yaml
+--plugin-config=prow/plugins.yaml
+--hmac-secret-file=path/to/hmac
+--github-token-file=path/to/github-token
+```
+
+## Usage
+Once you have a running server that manages github webhook events, generate an
+`hmac` token (same process as in [prow](../..)), and point a `phony` pull
+request event at it with the following:
+```
+bazel run //prow/cmd/phony -- --address=<server address> --hmac=<hmac token>
```

Comment:
Specifying the real default server address would be useful. It is `http://localhost:8888/hook`